cypress/e2e/homepage.cy.js

```diff
@@ -18,8 +18,47 @@ describe("the homepage", () => {
   });
 
   it("2048 Expert link redirects to its web page", () => {
-    cy.get('a[id="2048expert-btn"]').should("contain.text", "2048 Expert");
-    cy.get('a[id="2048expert-btn"]').click()
+    cy.get('a[id="a2048expert-btn"]').should("contain.text", "2048 Expert");
+    cy.get('a[id="a2048expert-btn"]').click()
     cy.url().should('eq', 'http://localhost:1234/2048-expert/index.html')
   });
+
+  it("should display a image in all buttons", () => {
+    cy.get('a[id="capidle-btn"]').find("img").should('be.visible');
+    cy.get('a[id="liedle-btn"]').find("img").should('be.visible');
+    cy.get('a[id="a2048expert-btn"]').find("img").should('be.visible');
+  });
+
+  it("should display the title image", () => {
+    cy.get('div[id="center-info"]').find('img[id="title-img"]').should('be.visible');
+  });
+
+  it("should display capidle details when button is hovered", () => {
+    cy.get('a[id="capidle-btn"]').realHover('mouse');
+    cy.get('div[id="center-info"]').find('div[id="capidle-details"]').should('be.visible');
+    cy.wait(1000)

```

Comment:
Do the tests pass without `cy.wait(1000)`? I think Cypress waits by default when the test is failing.

Reply:
It sometimes doesn't pass/detects it as visible. Doesn't explicitly state why but the most likely reason is because of the transition times. It consistently passes when `cy.wait(1000)` is added.